packages/date-picker/src/vaadin-date-picker.d.ts

```diff
@@ -79,7 +79,6 @@ export interface DatePickerEventMap extends HTMLElementEventMap, DatePickerCusto
  * Part name             | Description
  * ----------------------|--------------------
  * `toggle-button`       | Toggle button
- * `overlay-content`     | The overlay element
```

Comment:
I think we can also remove this for `vaadin-date-picker-light`

Reply:
Good point, done.